### PR TITLE
[Merged by Bors] - Improve eth1 fallback logging

### PR DESCRIPTION
**Pull request by pawanjay176**
## Issue Addressed

N/A

## Proposed Changes

There seemed to be confusion among discord users on the eth1 fallback logging
```
WARN Error connecting to eth1 node. Trying fallback ..., endpoint: http://127.0.0.1:8545/, service: eth1_rpc
```
The assumption users seem to be making here is that it is trying the fallback and fallback=endpoint in the log.

This PR improves the logging to be like
```
WARN Error connecting to eth1 node endpoint, endpoint: http://127.0.0.1:8545/, action: trying fallbacks, service: eth1_rpc
```

I think this is a bit more clear that the endpoint that failed is the one in the log.

### CODE REVIEW

File: beacon_node/eth1/src/service.rs [hunk 1]
```diff
@@ -215,16 +217,22 @@ async fn get_remote_head_and_new_block_ranges(
     if remote_head_block.timestamp + node_far_behind_seconds < now {
         warn!(
             service.log,
-            "Eth1 endpoint is far behind. Trying fallback ...";
+            "Eth1 endpoint is not synced";
             "endpoint" => endpoint,
+            "action" => "trying fallback";
```
Comment (blacktemplar) [inline beacon_node/eth1/src/service.rs:222]:
```suggestion
            "action" => "trying fallback",
```

**Review by blacktemplar — COMMENTED** [review]
Fixing the small type will hopefully make CI happy :), otherwise lgtm.

File: beacon_node/eth1/src/service.rs (hunk 1 above)
Comment (pawanjay176) [reply beacon_node/eth1/src/service.rs]: damn, I didn't even check it compiles before pushing! thanks :)

**Review by blacktemplar — APPROVED** [review]

**Review by AgeManning — APPROVED** [review]

**Pull request closed without merging**